app/src/main/java/com/onelist/ui/MainActivity.kt

```diff
@@ -50,58 +50,6 @@ class MainActivity : ComponentActivity() {
         }
     }
 
-    @Composable
-    fun ItemName() { //Example Form
-        //TODO Will be replaced with a list of items
-        var itemName by remember { mutableStateOf("") }
-        var quantity by remember { mutableStateOf("") }
-        var category by remember { mutableStateOf("") }
-        val context = LocalContext.current
-        Column {
-            OutlinedTextField(
-                value = itemName,
-                onValueChange = { itemName = it },
-                label = { Text(stringResource(R.string.ItemName)) },
-                modifier = Modifier.fillMaxWidth()
-            )
-            OutlinedTextField(
-                value = quantity,
-                onValueChange = { quantity = it },
```

Comment:
What's going on here?  Is this removed because it's not used?  If so... I agree.  But I'm not certain that is the case, without discussion.  :)

Reply:
It looks like I started a review when I did my comments instead of individual comments. I didn't understand the difference and it looks like the way I did it, I needed to finalized the review which I didn't know.